Directions.geojson

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "MultiPolygon",
+    "coordinates": [
+    "api_key": "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"
+    ]
+}

```

Comment:
Is this "Directions.geojson" an attempt to disguise your API key?  I don't think you are actually using it from here at all, are you?

I would strongly suggest that you do not check in your API key at all.

One option is to store the API key in a JSON file that you do not check-in to version control to. For that you'll need to parse some JSON:https://programmingwithswift.com/parse-json-from-file-and-url-with-swift/